build/web_server/Dockerfile

```diff
@@ -47,4 +47,4 @@ RUN npm run-script build
 COPY server/. /workspace/server/
 COPY shared/. /workspace/shared/
 WORKDIR /workspace
-CMD exec gunicorn --preload --timeout 1000 --bind :8080 -w 4 web_app:app
+CMD exec gunicorn --preload --timeout 1000 --bind :8080 -w 8 web_app:app
```

Comment:
Add a comment why 8?

Also was 8 necessary, or was it mostly the nl_server thing?

Reply:
Switching to 8 web app workers was a change I tried early on in the load testing process, so I'm honestly not sure if 4 would've also worked with the later changes. Since all the latest testing has been done with 8 web app workers and I don't see any big red flags for using that many, I think it's ok to just keep as 8?

Reply:
I see.  what is the mem-usage with 4 vs. 8?

Reply:
But yeah, I generally agree with the change (even if we don't have evidence of it alone improving things).

Reply:
I don't recall the exact mem-usage, but I remember the change did not require increasing the memory on the pods & the website-app container is limited to 2G of memory. Taking a look at the graphs, looks like current memory usage (20 pods, 8 workers each) is about 35G

Reply:
SG!